ftest/startup/itests.xml

```diff
@@ -1,13 +1,10 @@
-<project name="sample-selenium-tests" xmlns:nx="urn:nuxeo-build" xmlns:artifact="urn:nuxeo-artifact">
+<project name="sample-webdriver-tests" xmlns:nx="urn:nuxeo-build" xmlns:artifact="urn:nuxeo-artifact">
   <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build" />
   <taskdef resource="org/nuxeo/build/artifact/antlib.xml" uri="urn:nuxeo-artifact" />
   <taskdef resource="net/sf/antcontrib/antlib.xml" />
 
-  <!-- List of Selenium HTML test suites (without the ".html" file extension)  -->
-  <property name="suites" value="suite" />
-
-  <!-- Base distribution (nuxeo-cap is the default value) -->
-  <!-- <property name="wizard.preset" value="nuxeo-cap" /> -->
+  <!-- Tests are run against Nuxeo Server distribution, use 'mp.install' property to install any required MP -->
+  <!-- <property name="mp.install" value="nuxeo-jsf-ui" /> -->
```

Comment:
Maybe `ǹuxeo-jsf-ui` is not the best example of a package to install for a test on nuxeo-server?